src/cdk/testing/dom-helpers.ts

```diff
@@ -0,0 +1,9 @@
+export function getTextWithExcludedElements(element: Element, excludeSelector: string) {
```

Comment:
Much nicer. While you're at it, can you do the same thing for the other usages of `browser.executeScript`? They're much simpler functions, but would still be nice to have them be typed functions instead of strings